app/errors.py

```diff
@@ -0,0 +1,22 @@
+from abc import ABC, abstractmethod
+
+
+class VaccineError(Exception, ABC):
+    pass
+
+
+class NotVaccinatedError(VaccineError):
+
+    @abstractmethod
+    def __str__(self) -> str:
+        return "NotVaccinatedError"
```

Comment:
The `__str__` method in `NotVaccinatedError` should not be abstract since it is not intended to be overridden in subclasses. Remove the `@abstractmethod` decorator.